lib/user-interface/react/src/components/chatbot/Message.tsx

```diff
@@ -37,8 +38,30 @@ type MessageProps = {
 
 export default function Message ({ message, isRunning, showMetadata, isStreaming, markdownDisplay }: MessageProps) {
     const currentUser = useAppSelector(selectCurrentUsername);
-    const displayableMessage = message?.content + (!isStreaming && message?.metadata?.ragDocuments ? message?.metadata?.ragDocuments : '');
+
+    const getDisplayableMessage = (content: MessageContent, metadata?: LisaChatMessageMetadata) => {
+        if (Array.isArray(content)) {
+            return content.find((item) => item.type === 'text')?.text || '';
+        }
+        return content + (!isStreaming && metadata?.ragDocuments ? metadata.ragDocuments : '');
+    };
```

Comment:
This seems to be the same logic as `getDisplayableMessage` above, it just appends additional data. Maybe this should be a helper function that can be reused?